charts/alfresco-connector-msteams/Chart.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v2
 name: alfresco-connector-msteams
 description: A Helm chart for deploying Alfresco connector msteams service
 type: application
-version: 0.2.0
+version: 0.2.1-alpha.0
```

Comment:
you are adding new features which are retrocompat, so it's a minor version bump, not a patch